Mathlib/RingTheory/TwoSidedIdeal/Operations.lean

```diff
@@ -88,16 +88,40 @@ variable [NonUnitalRingHomClass F R S]
 
 /--
 Preimage of a two-sided ideal, as a two-sided ideal. -/
-def comap (I : TwoSidedIdeal S) : TwoSidedIdeal R where
-  ringCon := I.ringCon.comap f
+def comap : TwoSidedIdeal S →o TwoSidedIdeal R where
```

Comment:
Probably this should have simps now

Reply:
I am a bit hesitated to add a `simps` as it exposes the underlying `RingCon`, I hope this can be avoided by `mem_comap` below, I will add a seperate lemma for `monotone` part

Reply:
Maybe I should add `simps` attribute, then manually make them not `simp` lemma? What do you think @Ruben-VandeVelde